ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/FSRemover.java

```diff
@@ -194,8 +210,17 @@ private LockRequest createLockRequest(String dbName, String tableName, String pa
     }
```

Comment:
`protected LockRequest createLockRequest` is called only once, and it is the only method calling the `private LockRequest createLockRequest`, so there is no need for two overloads.

Reply:
Done.